crates/tabby-inference/src/chat.rs

```diff
@@ -54,7 +54,9 @@ impl ExtendedOpenAIConfig {
         &self,
         mut request: CreateChatCompletionRequest,
     ) -> CreateChatCompletionRequest {
-        request.model = self.model_name.clone();
+        if request.model.is_empty() {
+            request.model = self.model_name.clone();
```

Comment:
maybe good to check request.model has to belong to `supported_models` here (if not, generate a warning and set request.model = self.model_name)

Reply:
needs add field `supported_models` to ExtendedOpenAIConfig to